test/themeroller.js

```diff
@@ -9,32 +9,36 @@ const dirname = path.dirname( fileURLToPath( import.meta.url ) );
 describe( "ThemeRoller", function() {
 	let theme;
 
-	beforeEach( async function() {
-		const baseThemeCss = await fs.readFile( `${ dirname }/fixtures/jquery-ui-1.12/base/theme.css`, "utf-8" );
-		const varsString = await fs.readFile( `${ dirname }/fixtures/vars/base.json`, "utf-8" );
-		const vars = JSON.parse( varsString );
-
-		theme = new ThemeRoller( baseThemeCss, vars );
-	} );
+	[ "1.12.1", "1.13.2" ].forEach( ( jQueryUiVersion ) => {
```

Comment:
This one is the main file to review. I recommend reviewing with whitespace changes turned off due to the change in indentation.